requirements.txt

```diff
@@ -2,4 +2,5 @@ wes-service
 pandas
 IPython==5.8.0
 future
-bravado
\ No newline at end of file
+bravado
+toil
```

Comment:
Ah yeah — good call. I'm using the WDL parser included with toil to introspect descriptor files and identify input parameters. Would rather not have such a big dependency, but don't have a better idea for now.